app/src/main/res/values/component_colors.xml

```diff
@@ -256,5 +256,7 @@
   <color name="component_color_topic_info_fragment_download_status_image_color">@color/color_palette_primary_color</color>
   <color name="component_color_topic_info_fragment_download_story_count_text_color">@color/color_palette_show_text_color</color>
   <color name="component_color_topic_info_fragment_download_story_size_text_color">@color/color_palette_show_text_color</color>
+  <!-- Topic Revision Activity -->
+  <color name="component_color_topic_revision_activity_background_color">@color/color_palette_topic_revision_background_color</color>
```

Comment:
`component_color_topic_revision_fragment_background_color `

Reply:
> `component_color_topic_revision_fragment_background_color `

Okay sure I'll do that.